handler/middleware.go

```diff
@@ -576,7 +579,6 @@ func writeBody(mod wazeroapi.Module, buf, bufLen uint32, w io.Writer) {
 // getStatusCode implements the WebAssembly host function
 // handler.FuncGetStatusCode.
 func (m *middleware) getStatusCode(ctx context.Context, results []uint64) {
-	_ = mustBeforeNextOrFeature(ctx, handler.FeatureBufferResponse, "get", "status code")
```

Comment:
For reviewer: IMOH getStatusCode should only be called after next, I  think it is safe to remove the check here, since users mostly don't get status code in handleRequest.